### ISSUE TITLE
Support for longer task log text

### ISSUE THREAD

**Issue by dashmage** [dhth/hours#8]
While adding a task log entry with longer length, part of the text ends up getting truncated. This also happens when viewing them with `hours log`. It would be nice if the text were to scroll somehow or spill over to an additional line. But I can also imagine this as a feature where you expect the user to be able to concisely describe the task log?

task log list with a long entry
![image](https://github.com/user-attachments/assets/2a253ea9-17dd-4a39-9c6d-bda6e3aea801)

output of `hours log`
![image](https://github.com/user-attachments/assets/86df8af5-8abc-4bf5-b4de-1cb1103c172b)

As an aside, is there any way to edit the task log once created? Similar to updating a task description when pressing`u`?

**Comment by dhth** [comment]
You are correct in assuming that task log comments are intended to be concise. I'll see if I can make a task log comment take more than 1 line, and still keep the UI visually consistent, but my hunch is that it might make pagination much more of a hassle (right now each entry takes the same height (2 lines), so paginating entries is straightforward). Doing this for the log output might be comparatively easier.

If it makes sense, I might increase the max length for a task log comment (make it a paragraph instead of a line), and add another view (accessible via a single key, say "d") which shows the full comment in a pager.

Support for updating a task log entry is being worked in #11. I've also removed the trimming of task log comments in the task log list view.
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/47d23094-f516-4545-802e-22d779518d17">

**Comment by dashmage** [comment]
> You are correct in assuming that task log comments are intended to be concise. I'll see if I can make a task log comment take more than 1 line, and still keep the UI visually consistent, but my hunch is that it might make pagination much more of a hassle (right now each entry takes the same height (2 lines), so paginating entries is straightforward). Doing this for the log output might be comparatively easier.

Totally understand, I think it's good practice either way to keep the task log comment short and descriptive. It would be great if we could have the longer line support just for the task log output (`hours log`) if that might be simpler for the occasional long entry.

> If it makes sense, I might increase the max length for a task log comment (make it a paragraph instead of a line), and add another view (accessible via a single key, say "d") which shows the full comment in a pager.

I do sometimes find myself wanting to write an additional line or 2 for some entries which need more context. So if implemented, this would be quite useful actually.

> Support for updating a task log entry is being worked in https://github.com/dhth/hours/pull/11. I've also removed the trimming of task log comments in the task log list view.

That's great. Thanks so much! I've been using `hours` for a while now and it's a super productive part of my workflow.

**Comment by dhth** [comment]
Support for longer task log comments has been added in [v0.4.0](https://github.com/dhth/hours/releases/tag/v0.4.0).
